src/VisualStudio/CSharp/Impl/Options/CSharpSettingsManagerOptionSerializer.cs

```diff
@@ -1,511 +0,0 @@
-                            this.Manager.SetValueAsync(UnindentLabels, 0, isMachineLocal: false);
-                            return true;
-                        }
-
-                    case LabelPositionOptions.OneLess:
-                        {
-                            this.Manager.SetValueAsync(FlushLabelsLeft, value: 0, isMachineLocal: false);
-                            this.Manager.SetValueAsync(UnindentLabels, value: 1, isMachineLocal: false);
-                            return true;
-                        }
-                }
-            }
-
-            // code style: use this.
-            if (optionKey.Option == CodeStyleOptions.QualifyFieldAccess)
-            {
-                return PersistStyleOption<bool>(Style_QualifyFieldAccess, value);
-            }
-            else if (optionKey.Option == CodeStyleOptions.QualifyPropertyAccess)
-            {
-                return PersistStyleOption<bool>(Style_QualifyPropertyAccess, value);
-            }
-            else if (optionKey.Option == CodeStyleOptions.QualifyMethodAccess)
-            {
-                return PersistStyleOption<bool>(Style_QualifyMethodAccess, value);
-            }
-            else if (optionKey.Option == CodeStyleOptions.QualifyEventAccess)
-            {
-                return PersistStyleOption<bool>(Style_QualifyEventAccess, value);
-            }
-
-            // code style: use var options.
-            if (optionKey.Option == CSharpCodeStyleOptions.UseImplicitTypeForIntrinsicTypes)
-            {
-                return PersistStyleOption<bool>(Style_UseImplicitTypeForIntrinsicTypes, value);
-            }
-            else if (optionKey.Option == CSharpCodeStyleOptions.UseImplicitTypeWhereApparent)
-            {
-                return PersistStyleOption<bool>(Style_UseImplicitTypeWhereApparent, value);
-            }
-            else if (optionKey.Option == CSharpCodeStyleOptions.UseImplicitTypeWherePossible)
-            {
-                return PersistStyleOption<bool>(Style_UseImplicitTypeWherePossible, value);
-            }
-
-            return base.TryPersist(optionKey, value);
-        }
-
-        private bool PersistStyleOption<T>(string option, object value)
-        {
-            var serializedValue = ((CodeStyleOption<T>)value).ToXElement().ToString();
-            this.Manager.SetValueAsync(option, value: serializedValue, isMachineLocal: false);
-            return true;
-        }
-
-        private static bool FetchStyleOption<T>(string typeStyleOptionValue, out object value)
-        {
-            if (string.IsNullOrEmpty(typeStyleOptionValue))
-            {
-                value = CodeStyleOption<T>.Default;
-            }
-            else
-            {
-                value = CodeStyleOption<T>.FromXElement(XElement.Parse(typeStyleOptionValue));
-            }
-
-            return true;
-        }
-    }
```

Comment:
all this red makes me so happy.  i'm really hoping it doesn't come back later :)